pages/managed-accounts/_id/index.vue

```diff
@@ -42,7 +41,7 @@ export default defineComponent({
         const { hasAlert } = useKyVerified()
         const { getStartOfMonth, getEndOfMonth } = useLuxon()
 
-        const filters: Ref<GetManagedAccountStatementRequest | null> = ref(null)
+        const filters = ref<GetManagedAccountStatementRequest>({})
```

Comment:
{} is an empty object, more appropriate to still cast {} as GetManagedAccountStatementRequest or add the proprerties of the interface with initial values